static/js/apps/eval_retrieval_generation/sxs/app.tsx

```diff
@@ -32,11 +33,14 @@ interface AppPropType {
 
 export function App(props: AppPropType): JSX.Element {
   const [user, setUser] = useState<User | null>(null);
-  const [docInfo, setDocInfo] = useState<{ left: DocInfo; right: DocInfo }>(
-    null
+  const [docInfos, setDocInfos] = useState<{ a: DocInfo; b: DocInfo }>(null);
+  const { setSessionQueryId, sessionQueryId } = useContext(SessionContext);
+  const { left, right } = getLeftAndRight(
```

Comment:
nit: can we make the variable naming a little more informational, maybe leftDocInfo/rightDocInfo

Reply:
Done.